boututils/datafile.py

```diff
@@ -473,14 +473,32 @@ def _bout_type_from_dimensions(self, varname):
         dims_dict = {
             ('t', 'x', 'y', 'z'): "Field3D_t",
             ('t', 'x', 'y'): "Field2D_t",
+            ('t', 'x', 'z'): "FieldPerp_t",
             ('t',): "scalar_t",
             ('x', 'y', 'z'): "Field3D",
             ('x', 'y'): "Field2D",
+            ('x', 'z'): "FieldPerp",
+            ('x'): "ArrayX",
```

Comment:
Should there not be a comma here?

```suggestion
            ('x',): "ArrayX",
```